my-app/src/app/shared/components/header/header.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeaderComponent } from './header.component';
+import { FakeLogoModule } from '../fake-logo/fake-logo.module';
+import { LogInButtonModule } from '../log-in-button/log-in-button.module';
+import { LogOutButtonModule } from '../log-out-button/log-out-button.module';
+
+
+
+@NgModule({
+  declarations: [HeaderComponent],
+  imports: [
+    CommonModule,
+    FakeLogoModule,
```

Comment:
do not create unnecessary dependencies between shared modules. Combine shared components in the smart component